oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/dto/security/login/LoginDetail.java

```diff
@@ -0,0 +1,67 @@
+package com.hp.ov.sdk.dto.security.login;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import com.hp.ov.sdk.dto.BaseModelResource;
+
+public class LoginDetail extends BaseModelResource{
```

Comment:
sorry I am not able to find attributes state and status in the response of Login Detail. Can you please more elaborate about this comment.

Reply:
After looking carefully to the API, it does mention about attributes like category, modified, eTag, created and uri but in the response when i am actually running the example, these attributes are not coming. So shall i still keep it or remove the base class extension.

Reply:
You should only extend the base class if **ALL** the attributes declared in the class exist in the object. Otherwise you should declare in the object. It is possible that they are not being displayed because they might be null. You can always compare the return using an external tool to execute rest calls like POSTMAN for example.